components/docker-auth/pkg/extension/acl_test.go

```diff
@@ -0,0 +1,310 @@
+/*
+ * Copyright (c) 2019 WSO2 Inc. (http:www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http:www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package extension
+
+import (
+	"database/sql"
+	"fmt"
+	"io"
+	"log"
+	"net"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+)
+
+var dbConnection *sql.DB
+
+func createConn() bool {
+	dbDriver := MYSQL_DRIVER
+	dbUser := "root"
+	dbPass := "mysql"
+	dbName := DB_NAME
+	host := "localhost"
+	port := "3308"
+	var err error
+	dbConnection, err = sql.Open(dbDriver, dbUser+":"+dbPass+"@tcp("+host+":"+port+")/"+dbName)
+	if err != nil {
+		fmt.Println("Error while connecting to the database")
+		os.Exit(ErrorExitCode)
+	}
+	err = dbConnection.Ping()
+	if err != nil {
+		return false
+	}
+	return true
+}
+
+func moveFiles(from, to string) {
+	source, err := os.Open(from)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Try to return an error rather exiting here

Reply:
Fixed by #178